terragrunt/aws/api/inputs.tf

```diff
@@ -0,0 +1,3 @@
+variable "oidc_exists" {
+  type = bool
+}
```

Comment:
You should be able to get rid of this variable now since OIDC role creation is now handled in the `cds-snc/aft-account-requests` repo.

Reply:
Removed the reference and file.